openshift/templates/issuer-api/issuer-api-deploy.a2a.param

```diff
@@ -1,28 +1,28 @@
 #=========================================================
 # OpenShift template parameters for:
 # Component: .
-# Template File: ../openshift/templates/issuer-api/issuer-api-deploy.json
+# Template File: ../openshift/templates/issuer-kit-api/issuer-kit-api-deploy.json
 #=========================================================
-NAME=api
-APP_NAME=issuer
+NAME=api-a2a
+APP_NAME=issuer-a2a
 APP_GROUP=a2a
-SUFFIX=-a2a
-ROLE=controller
-IMAGE_NAMESPACE=nnfbch-tools
-NAMESPACE_NAME=nnfbch
+SUFFIX=
```

Comment:
Please update to use the `SUFFIX` for `-a2a`.

Reply:
updated suffixes for a2a